Analyzer/test/write_fit_input.py

```diff
@@ -227,26 +227,47 @@ def writeDataCard(self, name):
-            "SYY_850" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_850_SYY.root", label="SYY_850", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-
-            "SHH_350" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_350_SHuHd.root", label="SHH_350", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "SHH_450" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_450_SHuHd.root", label="SHH_450", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "SHH_550" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_550_SHuHd.root", label="SHH_550", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "SHH_650" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_650_SHuHd.root", label="SHH_650", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "SHH_750" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_750_SHuHd.root", label="SHH_750", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "SHH_850" : info.DataSetInfo(basedir=basedir, fileName="2016_stealth_stop_850_SHuHd.root", label="SHH_850", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-
-            "RPV_350" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_350.root",         label="RPV_350", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_450" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_450.root",         label="RPV_450", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_550" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_550.root",         label="RPV_550", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_650" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_650.root",         label="RPV_650", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_750" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_750.root",         label="RPV_750", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
-            "RPV_850" : info.DataSetInfo(basedir=basedir, fileName="2016_rpv_stop_850.root",         label="RPV_850", processName="signal", process="0", rate=True, lumiSys="1.05", scale=options.scale),
+            "SYY_300" : info.DataSetInfo(basedir=basedir, fileName="2016_StealthSYY_2t6j_mStop-300.root", label="SYY_300", processName="signal", process="0", rate=True, lumiSys="1.05", scale=1.17424),
```

Comment:
Every place that you put these new numbers can you do: (newXSec/oldXSec)*options.scale.
This way we still have the option to scale all samples by some common number if we want. I think if that give you problems you might need to change the default options.scale to be equal to 1.0

Reply:
Please test that it works when you do this.

Reply:
I changed everything you requested and changed the default and tested it (I also tested using the -s option)